### ISSUE TITLE
Is w_norm a hyperparameter?

### ISSUE THREAD

**Issue by xuzhang5788** [shaoanlu/AdamW-and-SGDW#1]
Thank you for your implementation.
What do you mean that " (w is the normalized one)'" in your sentence "w_norm: designed weight decay factor (w is the normalized one)" 
In your example, you chose w_norm as 0.0025. I think it is a hyperparameter. Am I right?
